explainaboard/analysis/analyses.py

```diff
@@ -212,21 +213,30 @@ def perform(
                 unwrap_generator(metrics),
                 unwrap_generator(stats),
             ):
-                bucket_stats = metric_stat.filter(bucket_collection.samples)
-                metric_result = metric_func.evaluate_from_stats(
-                    bucket_stats,
-                    conf_value=conf_value,
-                )
+                # Samples may be empty when user defined a bucket interval that
+                # has no samples
+                if n_samples == 0.0:
+                    value = 0.0
+                    conf_low: Optional[float] = 0.0
+                    conf_high: Optional[float] = 0.0
```

Comment:
It looks setting these values to `None` is better, because any values have no meaning under empty buckets. the low/high bound can become None, but maybe we need to change the type of value to `float | None`.

Reply:
Thanks! I initialized the `conf`s to None.

I agree with initializing value to None as well, but it may involve more changes in both SDK and the web interface. I'll open an issue and create a separate PR for it. Does this sound good to you?